### PR TITLE
Add status messages to associate teachers task

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

**Comment by MaxWofford** [comment]
@zachlatta The goal of this it to get a sense of how far the rake task is to completing.

**Comment by MaxWofford** [comment]
Reassigning to @paked because team bandwidth.

File: lib/tasks/outreach/teacher_emails/associate_teachers.rake [hunk 1]
```diff
@@ -36,16 +36,21 @@ task associate_teachers: :environment do
...
     t[:stage_key] == teacher_lead_stage_key && t[:linked_box_keys].count.zero?
   end
 
   pool = Concurrent::FixedThreadPool.new(THREAD_COUNT)
-  teachers_to_associate.each do |teacher|
+  puts 'Starting to associate!'
+  total = teachers_to_associate.length
+  teachers_to_associate.each_with_index do |teacher, i|
     pool.post do
+      puts "Starting index \##{i} out of #{total}"
```
Comment (paked) [inline lib/tasks/outreach/teacher_emails/associate_teachers.rake]: Shouldn't you put a mutex on stdout?

**Review by paked — CHANGES_REQUESTED** [review]

File: lib/tasks/outreach/teacher_emails/associate_teachers.rake (hunk 1 above)
Comment (MaxWofford) [inline lib/tasks/outreach/teacher_emails/associate_teachers.rake]: @paked Just added.

**Comment by paked** [comment]
WOo

**Pull request merged**